src/main/java/dicerolling/PoolProcessor.java

```diff
@@ -50,29 +54,21 @@ private void preprocess() {
         for (String param : paramArray) {
             if (param.matches("-fsu=[1-9]\\d*")) {
                 nextDiceFacetMod = Integer.parseInt(param.substring(5));
-            }
-            else if (param.matches("-fsd=[1-9]\\d*")) {
+            } else if (param.matches("-fsd=[1-9]\\d*")) {
```

Comment:
I personally prefer if else statements like if() {

}
else {

}

Reply:
It is done